src/model.js

```diff
@@ -0,0 +1,6 @@
+import falcor from '@graphistry/falcor';
```

Comment:
@trxcllnt interesting, I updated this PR w/ a custom reducer and removed the `redux-falcor` component. Now it works and I will try to use your approach w/ epics. Do you have any example I can follow? thanks again.